bundles/mapping/mapmodule/plugin/zoombar/ZoomSlider.jsx

```diff
@@ -141,38 +141,40 @@ export const ZoomSlider = ({ changeZoom, zoom = 0, maxZoom, isMobile = false, ..
     return (
         <Container>
             <MapModuleButton
-                icon={<PlusIcon />}
+                icon={<StyledPlus />}
                 className='t_plus'
                 onClick={() => {
                     changeZoom(zoom < 100 ? zoom + 1 : 100);
                 }}
                 size='18px'
+                iconSize='12px'
                 noMargin
             />
-            {!isMobile && (
-                <ThemeProvider value={mapModule.getMapTheme()}>
-                    <ThemedSlider
-                        className='t_zoomslider'
-                        vertical={true}
-                        value={zoom}
-                        step={1}
-                        dots
-                        max={maxZoom}
-                        min={0}
-                        onChange={value => {
-                            changeZoom(value);
-                        }}
-                        {...rest}
-                    />
-                </ThemeProvider>
-            )}
+
+            <ThemeProvider value={mapModule.getMapTheme()}>
+                <ThemedSlider
```

Comment:
Does this now always render the slider? It was previously only rendered on large screens. Also the +/- buttons were larger on smaller screens.

Reply:
There is isMobile check before this: https://github.com/oskariorg/oskari-frontend/blob/develop/bundles/mapping/mapmodule/plugin/zoombar/ZoomSlider.jsx#L119